Tests/StoreKitUnitTests/StoreKitConfigTestCase.swift

```diff
@@ -128,3 +128,19 @@ private extension StoreKitConfigTestCase {
     }
 
 }
+
+@available(iOS 15.0, tvOS 15.0, watchOS 8.0, macOS 12.0, *)
+extension StoreKitConfigTestCase {
+
+    func createTransaction(
+        productID: String? = nil,
+        finished: Bool
+    ) async throws -> StoreTransaction {
+        return StoreTransaction(
+            sk2Transaction: try await self.simulateAnyPurchase(productID: productID,
```

Comment:
How about making `simulateAnyPurchase` return `StoreKit.VerificationResult` instead so we can extract the JWS too?